source/common/network/connection_impl.h

```diff
@@ -162,6 +177,10 @@ class ConnectionImpl : public ConnectionImplBase, public TransportSocketCallback
 
   // Network::ConnectionImplBase
   void closeConnectionImmediately() final;
+  void remoteCloseThroughFilterManager() {
+    closeThroughFilterManager(ConnectionCloseAction{ConnectionEvent::RemoteClose, true});
+  }
```

Comment:
I prefer only keep the `closeThroughFilterManager` method.

Reply:
done.